manifest.json

```diff
@@ -22,7 +23,7 @@
         "url": "https://frju365.yunohost.support"
     },
     "requirements": {
-        "yunohost": ">= 4.3.0"
+        "yunohost": ">= 11.0.9"
```

Comment:
```suggestion
        "yunohost": ">= 11.1.6"
```

Reply:
yes it is necessary since the bug was solved that was stopping us from having a sandbox domain setup correctly

Reply:
can you make the change ?